src/index.html

```diff
@@ -2,31 +2,64 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Document</title>
+
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
```

Comment:
The indentation seems to be inconsistent. According to the checklist, 2 spaces should be used for indentation. Ensure that all nested elements have a consistent 2-space indentation from their parent elements.